src/Http/Http/perf/Microbenchmarks/Microsoft.AspNetCore.Http.Microbenchmarks.csproj

```diff
@@ -6,13 +6,34 @@
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <RootNamespace>Microsoft.AspNetCore.Http</RootNamespace>
+    <PreserveCompilationContext>true</PreserveCompilationContext>
```

Comment:
We need to enable this to support passing assembly info to the Roslyn compilation instance at runtime.